src/components/header/Header.jsx

```diff
@@ -30,38 +30,44 @@ const Header = () => {
         </Link>
       </li>
       <div className="right-nav">
-        <div className="desktop-action-btn">
-          <div className="desktop-query">
-            <SearchIcon className="right-nav-img img-search"></SearchIcon>
-            <input
-              className="desktop-searchbar"
-              type="text"
-              placeholder="Search for products"
-            />
-          </div>
-          <NavLinkUser
-            svg={<UserIcon className="right-nav-img single-action-btn" />}
+        <div className="desktop-query">
+          <SearchIcon className="right-nav-img img-search"></SearchIcon>
+          <input
+            className="desktop-searchbar"
+            type="text"
+            placeholder="Search for products"
           />
+        </div>
+        <div className="desktop-action-btn">
+          {/* <div className="single-action-btn"> */}
+          <NavLinkUser svg={<UserIcon className="right-nav-img" />} />
+          {/* </div> */}
+          {/* <div className="single-action-btn"> */}
           <Link to="/wishlist">
             <NavLinkItems svg={<AddToWishlist className="right-nav-img" />} />
             <button className="btn-icon btn-icon-wishlist">
               {wishlist.length}
             </button>
```

Comment:
The css of btn-icon-wishlist needs to be changed.
You need to position it as per the parent element
What you're doing right now is : 
.btn-icon-wishlist {
    left: 111.7rem;
    top: 1.3rem;
}
Here, based on the screen size the left value will be different

On my screen it looks like this : 
![xx](https://user-images.githubusercontent.com/95222845/163531264-b0158158-1ec6-400c-a22d-0750d3df8800.png)